ocaml-lsp-server/src/document.mli

```diff
@@ -55,6 +55,9 @@ module Merlin : sig
 
   val with_pipeline_exn : ?name:string -> t -> (Mpipeline.t -> 'a) -> 'a Fiber.t
 
+  val with_configurable_pipeline_exn :
+    ?name:string -> config:Mconfig.t -> t -> (Mpipeline.t -> 'a) -> 'a Fiber.t
```

Comment:
I think we should document how configuration is handled in both `with_pipeline_exn` and `with_configurable_pipeline_exn`.

Reply:
Done in https://github.com/ocaml/ocaml-lsp/pull/1265/commits/055f23db04f988e34e136a34a402093d2250e16a but feel free to rephrase it (or suggest rephrase) if it does not fit your expectation.